docs/contributing.md

```diff
@@ -3,6 +3,18 @@ We'd love to accept your patches and contributions to this project. There are ju
 
 When you are ready to get started developing, see our [development guide](development/development.md) for how to get started!
 
+## Weekly status calls
+
+Join us weekly on Mondays at 10 AM Eastern time!
```

Comment:
Great stuff!

Just a couple of pros/cons notes about the timing: 
- 10am Eastern is 7am Western which might be too early for that part.
- Mondays are hard for everyone and so people are busy with their own work.

Those might be limiting factors for someone joining the community calls and "fun" stuff.

But I think it's great to start it the way it's proposed and maybe optimizing in the future (if there's a need).

The advantage is that doing it on Mondays would feel like a BitOps week planning community call prioritizing the development that might be more important at this point 👍